### DISCUSSION TITLE
Renovate config option to return release notes URL

### DISCUSSION THREAD

**Discussion by Anagabsoares** [renovatebot/renovate#26326] [Request Help] [answered]
### Tell us more.

Hi Renovate folks, 

Currently, when Renovate updates a merge request (MR) with release notes, it includes the entire release notes content within the MR itself. While this is generally helpful, we have noticed that in some cases, release notes can become quite long, causing the MR page to take a significant amount of time to load, sometimes up to a minute or more.

It would be great if Renovate could provide an option to return the release notes URL instead of the full release notes content. This way, users can click on the URL to access the release notes separately, without overloading the MR page with lengthy release notes.

Maybe something like this?

```
 "prBodyTemplate": "{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{url}}}{{{configDescription}}}{{{controls}}}{{{footer}}}"

```

Is there already a way to achieve this behavior, which I may have overlooked? 

Thank you!

**Comment by rarkins** [comment]
You should generally find links to the source URL and/or changelog URL in the PR body table already. Usually it's just to the source repository, where it's up to you to click through to "Releases" or a file like Changelog.md. Example: https://github.com/renovatebot/renovate/pull/26330

If you want something different to that then creating a mockup of what you mean would help us understand

**Reply by devurandom** [reply]
We just started using Renovate and initially set `group:allDigest` to prevent it from spamming us with several dozen PRs.  However, this creates a long list of updated dependencies, with some of them having very lengthy change logs / release notes / news, which exceeds the GitHub PR description limit that Renovate observes: "This PR body was truncated due to platform limits".

What I would like to see is a Renovate config option `prBodyReleaseNotes: links-only` (or similar) that would make Renovate list the changed dependencies, but instead of copying the complete change log for each version into the PR description, it would only insert a link to the changelog, for each dependency.

**Reply by jamietanna** [reply, marked as answer]
https://github.com/renovatebot/renovate/issues/45510
